modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,65 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline, resource level
+
+By default, policies are applied to the entire API. However, you can implement an additional level of granularity for policies, where access is controlled based on the a criteria, called resource-level policies. 
+
+In this level of granularity, policies are applied to only those request that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+//* SLA-based policies
+//Impose a limit specified by the level of access granted to the requesting application.The Rate Limiting policy is SLA-based.
+// Is SLA based a policy level granularity or a configuration level? because it doesn't change where the policy is applied, just the configuration ( a.k.a limit) - Eva??
+
+
+//[PA: We need to tell the customers how to work with SLA-tiers for policies right?]
+//SLA tiers for me should go somewhere else and link to all the documentation on contracts, request api access and those flows. - Eva?
+
+Resource level policy support includes RAML and HTTP-based APIs (since Mule 4). A policy applied at a resource level affects all HTTP methods, for example, PUT, GET, POST, or selected HTTP methods, within the resource.
```

Comment:
```suggestion
Beginning with Mule 4, resource-level policy support includes RAML and HTTP-based APIs. A policy applied at a resource level affects all HTTP methods within the resource: PUT, GET, POST, and selected HTTP methods:
```
colon points to the image that follows, which should illustrate this sentence.

Reply:
Is it clear in this sentence that RAML based APIs are supported in Mule 3?

Reply:
Is this clearer: "With Mule 4, resource-level policy support now includes HTTP-based APIs along with RAML APIs, as opposed to support for only RAML APIs in Mule 3." Done! Thanks!